example-conf.yml

```diff
@@ -382,6 +382,9 @@ plugins:
     # Servermaps plugin: displays network /map's from the PyLink server's perspective.
     #- servermaps
 
+    # ServProtect plugin: disconnects from network if a certain limit of
+    # kills/saves per X seconds is met, THINK: X ACTIONs in X seconds
```

Comment:
Remove the "THINK: " part, just "disconnects from network if a certain limit of kills/saves per X seconds is met." is fine.